lhotse/recipes/gigaspeech.py

```diff
@@ -0,0 +1,41 @@
+from collections import defaultdict
+from typing import Dict, Optional, Sequence, Union
+
+from lhotse.audio import Recording, RecordingSet
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike, is_module_available
+
+if is_module_available('speechcolab'):
+    from speechcolab.datasets.gigaspeech import GigaSpeech
+else:
+    raise ImportError('To process the GigaSpeech corpus, please install optional dependency: pip install speechcolab')
```

Comment:
If I read this correctly, this makes `gigaspeech` hard requirement even when I don't intend to use this recipe. This module is imported in `recipes.__init__.py`. so this `ImportError` will be always raised if I don't have `gigaspeech`.

Reply:
Thanks for pointing this out. How about I check that module in the `prepare_gigaspeech()` function?

Reply:
How about
```python
if is_module_available('speechcolab'):
  import importlib
  speechcolab = importlib.import_module('speechcolab')
  GigaSpeech = speechcolab.datasets.gigaspeech.GigaSpeech
else:
  raise ImportError('.....')
```

Reply:
Thanks for noticing that. It should be a local import inside prepare_gigaspeech, then import lib is not needed. It works for other recipes and optional functionality.